### PR TITLE
Assignment 1 @dpeterson8 @Hex-Paladin

### PR DESCRIPTION
**Pull request by dpeterson8**
Page Link: https://gray-cliff-0ef225a10.3.azurestaticapps.net/

### CODE REVIEW

File: wordle-web/src/views/Game.vue [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<script setup lang="ts">
+</script>
```
Comment (roydunn19) [inline wordle-web/src/views/Game.vue:2]: `<script setup lang="ts"/>`

Comment (roydunn19) [reply wordle-web/src/views/Game.vue:2]: single line tags when no children

File: wordle-web/src/views/Game.vue [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        specimen book. It has survived not only five centuries, but also the leap into 
+        electronic typesetting, remaining essentially unchanged. It was popularised in
+         the 1960s with the release of Letraset sheets containing Lorem Ipsum passages,
+          and more recently with desktop publishing software like Aldus PageMaker including
+           versions of Lorem Ipsum.
+      </v-card-text>
+
+
+      <v-card-actions>
+        <RouterLink to="/">
```
Comment (roydunn19) [inline wordle-web/src/views/Game.vue:24]: Router link can be removed when v-btn has `to` prop

File: wordle-web/src/views/Game.vue [hunk 3]
```diff
@@ -0,0 +1,32 @@
+<script setup lang="ts">
+</script>
+
+<template>
+  <main>
+    <v-card class="mx-auto">
+      <v-card-item>
```
Comment (roydunn19) [inline wordle-web/src/views/Game.vue:7]: `<v-card-item>` not necessary. some output could be gotten without the extra layer

File: wordle-web/src/views/HomeView.vue [hunk 4]
```diff
@@ -0,0 +1,14 @@
+<script setup lang="ts">
+import TheWelcome from '../components/TheWelcome.vue'
+</script>
+
+<template>
+  <main>
+    <RouterLink to="/game">
```
Comment (roydunn19) [inline wordle-web/src/views/HomeView.vue:7]: Router link can be removed when v-btn has `to` prop

File: wordle-web/src/views/Game.vue
Comment (roydunn19) [inline wordle-web/src/views/Game.vue:1]: I would recommend making game its own component in the components folder then calling it in the game view file. this way the views folder is slimmed down and doesn't deal with much logic.

Comment (roydunn19) [reply wordle-web/src/views/Game.vue:1]: another cool thing you could do is move all jsx elements into a function in your script tag then call that function in your template tag
`const game = () => { JSX ELEMENTS HERE }`

**Review by mmwoodfo — COMMENTED** [review]
Good job!

**Pull request closed without merging**